DependencyInjection/IseWebSecurityExtension.php

```diff
@@ -26,8 +26,15 @@ public function load(array $configs, ContainerBuilder $container)
         );
         $loader->load('services.yaml');
         
+        $presets = Yaml::parseFile(__DIR__.'/../Resources/config/presets.yaml')['presets'];
```

Comment:
This seems a bit fragile. Does Symfony not provide a link to the config directory somewhere?

Reply:
I agree but, I've not really found a trivial way to implement it. This code, just above it 
```php
        $loader = new YamlFileLoader(
            $container,
            new FileLocator(__DIR__.'/../Resources/config')
        );
        $loader->load('services.yaml');
```
Sort of does that, although if I try `$loader->load('presets.yaml');` I'll need to create a loader service specifically for that type of resource, which I'm not sure is worth it given the format itself is no more complex than an associative array.

I won't shy away from it if you think it would be more robust to do so.